SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -188,10 +189,13 @@ if [[ ${REPO_TYPE} == "git" ]]; then
     REPO_PATH="${GIT_HOME}/$(basename $REPO_PATH)"
 fi
 
+_log "ENV_NAME:${ENV_NAME}"
 _log "REPO_PATH:${REPO_PATH#$HOME}"
 
-# Install a Jupyter kernel for the environment
-python -m ipykernel install --name "${ENV_NAME}" --display-name "Python (${ENV_NAME})" --prefix "${ENV_PATH}" | tee -a "${LOG_FILE}"
+# Install the same ipykernel that the Jupyter server uses
+pip install ipykernel==${IPYKERNEL_VERSION} | tee -a ${LOG_FILE}
```

Comment:
I would prefer to move the installation of ipykernel into the individual scripts and, in the case of conda-mamba, install ipykernel with mamba and not pip. Even if pip inside a conda environment can work, I think it's better that we use mamba to install packages that are available in coda repos. Here some people list reasons not to do pip inside conda:

https://www.reddit.com/r/learnpython/comments/q4u6ng/how_bad_is_installing_pip_packages_in_a_conda/

Reply:
From the tests I did, I noticed conda has ipykernel already. Do you still want to keep it for consistency purposes or do you think it could be avoided to get the environment created faster?

Reply:
Since we enforce a certain ipykernel version, we need to install ipykernel explicitly.

Reply:
Done.